### ISSUE TITLE
My notes about your version for chapters 5 and 6.

### ISSUE THREAD

**Issue by Hexstream** [robert-strandh/CLOS-MOP-HTML#3]
https://www.hexstreamsoft.com/tmp/clos-mop-chapter-5-notes-about-robert-strandh-version.html has my notes about your version for chapter 5 in a fairly raw form, I hope this will be sufficiently palatable to you.

**Comment by robert-strandh** [comment]
I think I can work with these notes.  Thanks!

Should I copy them, or do you think your URL will remain valid?

**Comment by Hexstream** [comment]
Great!

Policy-wise, stuff in "/tmp/" on my sites is generally not guaranteed to remain available, although empirically most of it does tend to remain available for an extended period. One of the biggest reasons for having a "/tmp/" folder is that I can just throw stuff in there with no guarantees, I can theoretically just delete anything anytime, or it might get lost because of whatever reason and it doesn't matter too much.

I really don't want to start giving guarantees about files in "/tmp/", but I can somewhat accommodate you in this particular case, if that's useful.

This particular "/tmp/" folder happens to be hosted on my S3 "bucket" for www.hexstreamsoft.com (which means that outside of extraordinary circumstances, it will stay there until I decide to delete it), and I also have a copy of the file in my "temporary" git repo for clos-mop.hexstreamsoft.com (which I have copies of on 2 separate machines), which I will completely wipe out when I'm ready to publish by creating a new fresh repo with one "Initial commit". (I really love burning my notes!)

So in this instance, you can rely on the file staying there until I'm ready to publish `clos-mop.hexstreamsoft.com`, which will likely still take a few weeks (I actually have fairly big plans beyond what you've already seen in the screenshots). I could arrange for hosting the file in a more long-term location, but I don't want to commit to holding on to this file indefinitely, and I also don't want to commit this file to a non-temporary git repo, where the file will live in the commit history forever, after all the point of finally reporting these issues to you is that I can stop caring about them. I could also host this in a "gist", but I'm not too familiar with gists, and I also don't want to maintain this gist in my account indefinitely anyway.

---------------------------------------------------------

Anyway, sorry for the extensive background information, but basically I would much prefer if you either "processed" this file fast enough that its existence or non-existence quickly becomes irrelevant, or maybe you could import it into one of your own repos somewhere, or you could create a "gist" for it, or otherwise keep a copy somewhere on your own infrastructure. The sooner I can stop caring about this file, the better. :)

**Comment by Hexstream** [comment]
My notes about your version for chapter 6 are now also available. Enjoy!

https://www.hexstreamsoft.com/tmp/clos-mop-chapter-6-notes-about-robert-strandh-version.html

**Comment by Hexstream** [comment]
Oops, sorry, I just noticed that I accidentally wiped out `https://www.hexstreamsoft.com/tmp/*` a bit earlier than I had planned. (I had forgotten to update my `aws s3 sync --delete` command to `--exclude 'tmp/*'`.)

I guess this is a good opportunity to verify if you've already downloaded my notes for chapter 5 and 6. Given what I said above about the transience of `/tmp/`, I think you did, else I'll just upload the files again.

----------------------------------

Incidentally, I noticed that I had accidentally wiped out `/tmp/` while uploading a new screenshot of my Chapter 6, so here it is, just for fun (scroll down a bit for some fun "merging", "Generated", navigation, and "diff" stuff, you'll see what I mean): https://www.hexstreamsoft.com/tmp/wip/clos-mop-chapter-6_2017-07-01.png
